sktime/forecasting/fbprophet.py

```diff
@@ -41,6 +41,8 @@ class Prophet(_ProphetAdapter):
         Dict with args for Prophet.add_country_holidays().
         Dict can have the following keys/values:
             country_name: Name of the country, like 'UnitedStates' or 'US'
+    fit_kwargs: dict or None, default=None
+        Dict with args for Prophet.fit().
```

Comment:
perhaps explain better that these do not include data arguments.
We should also add what we know about valid values.